apps/commudle-admin/src/app/feature-modules/public-community/components/events/events.component.html

```diff
@@ -1,77 +1,100 @@
 <nb-card [nbSpinner]="eventLoader">
   <nb-card-body>
-    <div class="events d-flex flex-wrap justify-content-center">
-      <a
-        *ngFor="let event of events"
-        [routerLink]="['/communities', event.kommunity_slug, 'events', event.slug]"
-        class="event-clickable"
-      >
-        <nb-card [ngClass]="{ upcoming: moment(event.end_time) > moment() && event.header_image_path }" class="event">
-          <nb-badge
-            *ngIf="moment(event.start_time) > moment()"
-            position="top left"
-            status="success"
-            text="New"
-          ></nb-badge>
-          <nb-card-body class="d-flex">
-            <div class="header-image d-flex align-items-center justify-content-center">
-              <span>
-                <app-badge
-                  *ngIf="moment(event.start_time) <= moment() && moment(event.end_time) >= moment()"
-                  [color]="'rgb(255 45 45)'"
-                  [text]="'Live'"
-                ></app-badge>
-              </span>
-              <img
-                *ngIf="event.header_image_path"
-                alt="{{ event.name }}"
-                src="{{ event.header_image_path }}"
-                title="{{ event.name }}"
-              />
-              <nb-icon *ngIf="!event.header_image_path" icon="link" status="primary"></nb-icon>
-            </div>
-            <div class="details">
-              <h5>
-                <nb-icon
-                  *ngIf="moment(event.end_time) < moment()"
-                  icon="checkmark-circle-2"
-                  nbTooltip="Completed"
-                  status="success"
-                ></nb-icon>
-                <span class="event-name">
-                  {{ event.name }}
-                  <span *ngIf="event.kommunity_id !== community.id" class="collaboration">(Collaboration)</span>
+    <div class="events">
```

Comment:
try to use section , in upcoming and past events

Reply:
Done